README.md

```diff
@@ -28,9 +28,9 @@ Currently, it **must** be built using a nightly release of the Rust compiler.
 Nightly releases can be found at the bottom of the
 [Rust install page](http://www.rust-lang.org/install.html)
 or installed using the `--channel=nightly` option to
-[rustup](https://github.com/rust-lang/rustup).
+[rustup.sh](https://github.com/rust-lang/rustup).
```

Comment:
You've linked the wrong rustup project here. It should be `[rustup.rs](https://github.com/rust-lang-nursery/rustup.rs)` as below.